airflow/cli/cli_parser.py

```diff
@@ -73,21 +73,25 @@ def error(self, message):
         self.exit(2, f'\n{self.prog} command error: {message}, see help above.\n')
 
 
+# Used in Arg to enable `None' as a distinct value from "not passed"
+_UNSET = object()
+
+
 class Arg:
     """Class to keep information about command line argument"""
-    # pylint: disable=redefined-builtin
-    def __init__(self, flags=None, help=None, action=None, default=None, nargs=None,
-                 type=None, choices=None, required=None, metavar=None):
+    # pylint: disable=redefined-builtin,unused-argument
+    def __init__(self, flags=_UNSET, help=_UNSET, action=_UNSET, default=_UNSET, nargs=_UNSET, type=_UNSET,
+                 choices=_UNSET, required=_UNSET, metavar=_UNSET):
         self.flags = flags
-        self.help = help
-        self.action = action
-        self.default = default
-        self.nargs = nargs
-        self.type = type
-        self.choices = choices
-        self.required = required
-        self.metavar = metavar
-    # pylint: enable=redefined-builtin
+        self.kwargs = {}
+        for k, v in locals().items():
+            if v is _UNSET:
+                continue
+            if k in ("self", "flags"):
+                continue
+
+            self.kwargs[k] = v
+    # pylint: enable=redefined-builtin,unused-argument
```

Comment:
Can we add a test for this, e.g adding a new argument with a false default

Reply:
Good call.

Reply:
Done.